astropy/io/ascii/tests/test_read.py

```diff
@@ -1352,7 +1353,13 @@ def test_guessing_file_object():
 def test_pformat_roundtrip():
     """Check that the screen output of ``print tab`` can be read. See #3025."""
     """Read a table with empty values and ensure that corresponding entries are masked"""
-    table = "\n".join(["a,b,c,d", "1,3,1.11,1", "2, 2, 4.0 , ss "])
+    table = dedent(
```

Comment:
Here, the change is for the better, though I'd do it as in the above example myself (i.e., items on separate lines, but kept as single strings).

Reply:
Done.